### ISSUE TITLE
Zero pause freezes ATL transform 

### ISSUE THREAD

**Issue by via-my** [renpy/renpy#3520]
Issue with Ren'Py Nightly from May 16.

I have a code that animates menu items with pause. 
If pause equals 0 transform completely freezes.

```
transform preferences_transform(pause=0, alpha=1.0):
        xoffset 75
        alpha .0
        pause pause
        easein_back .3 xoffset 0 alpha alpha
```

Adding any value to the pause (like pause + 0.000001) makes it work.

**Comment by Gouvernathor** [comment]
This was possibly fixed by #3514.

**Comment by via-my** [comment]
Yes! Now it works
